CMakeLists.txt

```diff
@@ -741,8 +741,8 @@ if(BUILD_TESTS)
     NAME js_e2e_logging_raft
     PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_logging.py
     CONSENSUS cft
-    ADDITIONAL_ARGS --js-app-script
```

Comment:
It seems that we're not using `--js-app-script` for anything anymore and that the app bundle should be used instead as it exports the ccf module for access to KV and typing?

Reply:
Agreed - planning to do this in a future PR (deprecate rather than remove if possible, for `--js-app-script`, `set_js_app` proposals, &c).